holmes/plugins/toolsets/coralogix/utils.py

```diff
@@ -80,54 +76,81 @@ def normalize_datetime(date_str: str) -> str:
         date = datetime.fromisoformat(date_str.rstrip("Z"))
         # Ensure the timestamp has the maximum resolution (nanoseconds)
         normalized_date_time = date.strftime("%Y-%m-%dT%H:%M:%S.%f")
+        logging.error(f"normalized datetime {date_str} => {normalized_date_time}")
```

Comment:
By mistake?

Reply:
I am debugging compatibility issues with older python versions. I will remove these debug lines once it's fixed.